src/v2/guide/typescript.md

```diff
@@ -137,3 +137,63 @@ export default class MyComponent extends Vue {
 ```
 
 With this syntax alternative, our component definition is not only shorter, but TypeScript can also infer the types of `message` and `onClick` without explicit interface declarations. This strategy even allows you to handle types for computed properties, lifecycle hooks, and render functions. For full usage details, see [the vue-class-component docs](https://github.com/vuejs/vue-class-component#vue-class-component).
+
+## Declaring Types of Vue Plugins
+
+Plugins may augment Vue's global/instance properties, component options and so on. For that case, augmenting type declaration is also needed to use plugins in TypeScript. Fortunately, there is a feature to augment an existing types that is called [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).
```

Comment:
`augment` appears three times in this paragraph, which reads quite wordy. Also `augment` is rather an abstract term, so it is quite hard for TypeScript new comers. 

I would suggest change some verbs:

> Plugins may add Vue's global/instance properties, component options and so on. For that case, type declaration also needs updates to make plugins code compile in TypeScript.

`add` and `update` explains what `augment` does before we introduce the TS feature. I think this should be clearer for ordinary Vue users.